content/project/projects/libraries/ouroboros/contents.lr

```diff
@@ -8,12 +8,10 @@ languages: py
 ---
 platforms: independent
 ---
-short_description: A standalone, pure Python implementation of the Python Standard Library.
+short_description: A standalone, pure Python implementation of the Python Standard Library. This allows Python default libraries to be used portably on any implementation of the Python virtual machine---not requiring CPython.
 ---
 description: A standalone, pure Python implementation of the Python Standard Library.
 ---
-help_required: 
+help_required: Expansion of unit testing. 
 ---
-pun: The ouroboros is an ancient symbol depicting a serpent eating its own tail.
----
-incomplete: yes
```

Comment:
The incomplete refers to the website page itself, not the project; in this case, the changes flesh out enough to make it "complete".